docs/declarative-customization/view-formatting.md

```diff
@@ -262,52 +556,363 @@ The following image shows a customized card in Gallery layout:
 }
 ```
 
+### Build custom group headers
+
+You can use `groupProps` to format group headers with flexibility to add grouped column's data, display name and item count. You can also add group aggregates in the group headers.
+
+#### Example: Color coded group header
+
+In the example below we have gallery with group headers formatted according to column metadata.
+![Employee gallery grouped by City with formatted group header](../images/employee-formatted-group-header-gallery-layout.png)
+
+Note that gallery card formatter is skipped in the below JSON for simplicity.

```

Comment:
There is a particular format to put a box around and change bgc for notes to highlight, can we do that ?